packages/components/src/icon-button/index.js

```diff
@@ -1,6 +1,7 @@
 /**
  * WordPress dependencies
  */
+import deprecated from '@wordpress/deprecated';
```

Comment:
I’m on mobile so I can’t do a proper check. Is README file removed? It should as our docs generator will still keep it exposed. Maybe we could remove the folder completely and put the file in deprecated subfolder?

Reply:
Good call